src/main/scala/nl.knaw.dans.easy.validatebag/rules/metadata/package.scala

```diff
@@ -237,9 +240,39 @@ package object metadata extends DebugEnhancedLogging {
     else Try(fail(s"Point with only one coordinate: ${ point.text.trim }"))
   }
 
+  def archisIdentifiersHaveAtMost10Characters(t: TargetBag): Try[Unit] = {
+    trace(())
+    for {
+      ddm <- t.tryDdm
+      identifiers = getArchisIdentifiers(ddm)
```

Comment:
I'd say you are not getting the identifiers but the identifier-elements. I had a [similar comment for @vesaakerman](https://github.com/DANS-KNAW/easy-validate-dans-bag/pull/35#discussion_r221947974).

Reply:
You are right. Changed it. See da09285.